modules/installation-vsphere-infrastructure.adoc

```diff
@@ -41,8 +37,3 @@ you install {product-title}.
 ====
 You must ensure that the time on your ESXi hosts is synchronized before you install {product-title}. See link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vcenterhost.doc/GUID-8756D419-A878-4AE0-9183-C6D5A91A8FB1.html[Edit Time Configuration for a Host] in the VMware documentation.
 ====
-
-[IMPORTANT]
-====
-A limitation of using VPC is that the Storage Distributed Resource Scheduler (SDRS) is not supported. See link:https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/faqs.html[vSphere Storage for Kubernetes FAQs] in the VMware documentation.
-====
```

Comment:
Seems we still not support SDRS, is this documented somewhere else?

Reply:
storage vmotion is basically same as SDRS.

Reply:
I see. Thanks!